src/setup/cortx_deploy

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Log Levels:
+DEBUG=1
+INFO=2
+WARNING=3
+ERROR=4
+FATAL=5
+DEFAULT_LOG_LEVEL=${DEFAULT_LOG_LEVEL:-$INFO}
+
+function log {
+    LEVEL=$1; shift
+    LOG_LEVEL=${DEBUG:-$DEFAULT_LOG_LEVEL}
```

Comment:
Since you changed the command line variable from DEBUG to LOG_LEVEL you need to use the same here. 
LOG_LEVEL=${LOG_LEVEL:-$DEFAULT_LOG_LEVEL}